lib/vagrant-service-manager/services/open_shift.rb

```diff
@@ -6,12 +6,14 @@ class OpenShift < ServiceBase
       def initialize(machine, env)
         super(machine, env)
         @service_name = 'openshift'
-        @extra_cmd = build_extra_command
       end
 
       def execute
         if service_start_allowed?
-          command = "#{@extra_cmd} sccli openshift"
+          command = 'sccli openshift'
```

Comment:
> Then I give an additional vote to renaming to openshift.rb. Relying on the filename is wrong.

This is general pattern in Ruby to have such names. Ruby on Rails codebase is full of this pattern which follows convention over configurations. It would have been difficult if they didn't use this pattern.